themes/scss/widgets/themes/light.scss

```diff
@@ -217,42 +219,40 @@ $chartMiniHighlight: rgba(#333, 1);
     }
   }
 
-
   //Charts
   .extent {
     stroke: $link;
   }
 
-  .CDB-Chart-bar {
+  .CDB-Chart-bar--green {
     fill: $highlighted;
   }
-
-  .CDB-Chart-bar.is-highlighted {
+  .CDB-Chart-bar--green.is-highlighted {
     fill: $darkHighlighted;
   }
-
+  .CDB-Chart-bar--orange {
+    fill: $highlightedAlternative;
+  }
+  .CDB-Chart-bar--orange.is-highlighted {
```

Comment:
+1 to use modifier, although :-1: to use the color as the name… why not have something semantic, like `CDB-Chart--timeSeries`?